pkg/workflow_execution.go

```diff
@@ -1463,3 +1463,24 @@ func (c *Client) getWorkflowExecutionAndTemplate(namespace string, uid string) (
 
 	return
 }
+
+func (c *Client) DeleteWorkflowExecution(namespace, uid string) error {
+	err := c.ArgoprojV1alpha1().Workflows(namespace).Delete(uid, nil)
+	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
+			return nil
+		}
+		return err
+	}
+	return nil
+}
+
+func (c *Client) DeleteWorkflowExecutionDb(workflowUid string) error {
+	query := `DELETE FROM workflow_executions
```

Comment:
Can this query be rewritten using Squirrel?

Reply:
Refactored